selfdrive/car/mazda/interface.py

```diff
@@ -22,14 +22,14 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=None):
     ret.safetyConfigs = [get_safety_config(car.CarParams.SafetyModel.mazda)]
     ret.radarOffCan = True
 
-    ret.dashcamOnly = candidate not in (CAR.CX9_2021,)
+    ret.dashcamOnly = candidate not in (CAR.CX5_2022, CAR.CX9_2021)
```

Comment:
is it time to blacklist the dashcam only ones instead? :)

Reply:
Not yet! 4 vs 2.